homeassistant/components/nuki/lock.py

```diff
@@ -58,6 +59,11 @@ async def async_setup_entry(hass, entry, async_add_entities):
             vol.Optional(ATTR_UNLATCH, default=False): cv.boolean,
         },
         "lock_n_go",
+        "set_continuous_mode",
+        {
+            vol.Optional(ATTR_ENABLE, default=False): cv.boolean,
```

Comment:
I think we should not set a default. It makes it all more readable later in the automation and it prevents turning it off by mistake.